src/lib/relink_private.js

```diff
@@ -24,9 +24,9 @@ module.exports = function relinkPrivateKeys(toContainer, fromContainer) {
         var fromVal = fromContainer[k];
         var toVal = toContainer[k];
 
-        if(toVal === fromVal) {
-            continue;
-        }
+        if(toVal === fromVal) continue;
+        if(k !== '_categories' && toContainer.matches) continue;
```

Comment:
Wait, what's going on here? Am I missing something or won't this skip everything *except* `_categories`, when `matches` is defined?

Maybe what this is saying is we actually *can* skip every private key for `matches` axes?

Reply:
Oops! Good eye!
Fixed in https://github.com/plotly/plotly.js/pull/4832/commits/2295e505b7b8f52b31e23d0e4c7bdcd3f27db4e2.